lisp/ess-r-mode.el

```diff
@@ -443,81 +443,6 @@ before ess-site is loaded) for it to take effect.")
   "List of R-versions found from `ess-r-versions' on the system.")
 (define-obsolete-variable-alias 'ess-r-versions-created 'ess-r-created-runners "2018-05-05")
 
-(defun ess-r-s-define-runners ()
```

Comment:
You have removed this, but where did you put it back? I don't see it in this diff.

Reply:
I didn't put it back. I don't think it's needed now. R runners are created when loading ess-r-mode and S runners are created when loading ess-sp6-d, so there's no need for this function anymore (I think).

Reply:
Arh. Right. I assumed it was renamed into `ess-r-define-runners`, but of course, that's a different function.